libafl/src/executors/command.rs

```diff
@@ -494,6 +493,76 @@ where
     }
 }
 
+/// The shared implementation for children with stdout/stderr/timeouts.
+pub trait ChildrenArgs: Sized {
+    /// The timeout of the children
+    fn timeout_ref(&self) -> &Duration;
+    /// The mutable refernce of the timeout of the children
+    fn timeout_mut(&mut self) -> &mut Duration;
+
+    /// The stderr handle of the children
+    fn stderr_ref(&self) -> &Option<Handle<StdErrObserver>>;
+    /// The mutable stderr handle of the children
+    fn stderr_mut(&mut self) -> &mut Option<Handle<StdErrObserver>>;
```

Comment:
What's an optional mutable handle supposed to do? Do we need this?
Should the name be `stderr_handle_mut` instead?

Reply:
Capturing stdout/stderr is big overhead and should be optional.